src/main/java/duke/Storage.java

```diff
@@ -0,0 +1,107 @@
+package duke;
+
+import java.io.File;
+import java.io.FileWriter;
+
```

Comment:
Is this line break meant to separate the imports? I believe it won't be necessary as the import on line 4 and line 6 are part of the same group 👍